src/js/grades/Grade.ts

```diff
@@ -24,6 +24,18 @@ const gradeContextToGradeScales = {
     aid: GradeScales.FRENCH,
     snow: GradeScales.FRENCH, // is this the same as alpine?
     ice: GradeScales.FRENCH // is this the same as alpine?
+  },
+  AU: {
```

Comment:
@musoke I copied this AU context from the backend.  Whenever it goes in `sandbag` we can change it.

Reply:
Ok!  I'll move that over once we know this is working